libethereum/BlockChainSync.cpp

```diff
@@ -177,6 +177,12 @@ BlockChainSync::~BlockChainSync()
 	abortSync();
 }
 
+void BlockChainSync::onBlockImported(BlockHeader const& _info)
+{
+	string number = dev::toString(_info.number());
+	m_lastImportedBlock = atoi(number.c_str());
```

Comment:
`m_lastImportedBlockHash` needs to be updated every time `m_lastImportedBlock` is updated